scripts/install.sh

```diff
@@ -989,6 +1000,7 @@ fi
 echo 'Changing ownership for config and storage'
 chown -R "${SYSTEM_USER}":"${SYSTEM_USER}" "${CONFIG_PATH}" "${FILE_STORAGE}"
 
+
```

Comment:
Just like newlines?

Reply:
left over from when i moved the env directory chown further up